Wikipedia/Code/UIImageView+WMFImageFetchingInternal.m

```diff
@@ -158,12 +158,23 @@ - (void)wmf_setImage:(UIImage *)image
              success:(WMFSuccessHandler)success {
     NSAssert([NSThread isMainThread], @"Interaction with a UIImageView should only happen on the main thread");
 
-    CGRect faceBounds = [faceBoundsValue CGRectValue];
     if (detectFaces) {
-        CGFloat faceArea = faceBounds.size.width * faceBounds.size.height;
-        if (!CGRectIsEmpty(faceBounds) && (faceArea >= 0.05)) {
-            [self wmf_cropContentsByVerticallyCenteringFrame:[image wmf_denormalizeRect:faceBounds]
-                                         insideBoundsOfImage:image];
+
+        CGRect faceBounds = [faceBoundsValue CGRectValue];
+        if (!CGRectIsEmpty(faceBounds)) {
+            
+            CGFloat faceArea = (image.size.width * faceBounds.size.width) * (image.size.height * faceBounds.size.height);
+            CGFloat imageArea = image.size.width * image.size.height;
+            CGFloat faceProportionOfImage = faceArea / MAX(imageArea, 0.0000001);
+            // Reminder: "0.0178" of the area of a 640x640 image would be roughly an 85x85 area.
+            BOOL faceIsBigEnough = (faceProportionOfImage >= 0.0178);
```

Comment:
The new proportional value of `0.0178` was chosen by simply increasing it until the test passed.